src/core/metadata/parameters/verifier.rs

```diff
@@ -8,6 +9,31 @@ use serde_json::{Map, Value as Json};
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct VpFormats(pub ClaimFormatMap);
 
+impl VpFormats {
+    /// Returns a boolean to denote whether the format and cryptosuite provided
+    /// are supported in the VP formats.
+    ///
+    /// NOTE: This method is interested in the cryptosuite of the claim format
+    /// payload and not the claim format designation.
+    ///
+    /// For example, the cryptosuite would need to match one of the `alg`
+    /// values in the claim format payload.
+    pub fn supports_cryptosuite(
+        &self,
+        format: &ClaimFormatDesignation,
+        cryptosuite: &String,
```

Comment:
There's some mixed semantics here. Different formats use either `proof types` (AKA `cryptosuite`) or `alg`  (AKA signature algorithm) to identify how the credential/presentation is secured. Conflating them all as `cryptosuite` is confusing as that is specifically a W3C VC term.
```suggestion
    /// Returns a boolean to denote whether a particular pair of format and security method
    /// are supported in the VP formats. A security method could be a JOSE algorithm, a COSE
    /// algorithm, a Cryptosuite, etc.
    ///
    /// NOTE: This method is interested in the security method of the claim format
    /// payload and not the claim format designation.
    ///
    /// For example, the security method would need to match one of the `alg`
    /// values in the claim format payload.
    pub fn supports_security_method(
        &self,
        format: &ClaimFormatDesignation,
        security_method: &String,
```